spec/lib/spotlight/dor/indexer_spec.rb

```diff
@@ -498,4 +498,87 @@
     end # each
   end # add_folder_name
   # rubocop:enable Metrics/LineLength
+
+  # rubocop:disable Metrics/LineLength
+  describe '#add_object_full_text' do
+    let(:expected_text) do
+      'SOME full text string that is returned from the server'
+    end
+    let(:full_file_path) do
+      'https://stacks.stanford.edu/file/oo000oo0000/oo000oo0000.txt'
+    end
+    let(:public_xml_with_feigenbaum_full_text) do
```

Comment:
I would say if these vars only pertain to a single test, put them inside the test block?   I tend to use let and let! for stuff used in multiple tests.   Note:  let!  is how you can keep something instantiated across tests, so the first two could be let!  if you want.